app/views/stories/new.html.erb

```diff
@@ -24,26 +24,26 @@
           <%= f.label :body %>
           <%= f.text_area :body, rows: 16, class: 'form-control', placeholder: "Story text" %>
         </div>
-        <div class="form-group">
-          <%= f.label :price %>
-          <%= f.number_field :price, class: 'form-control', placeholder: "0" %>
-        </div>
-        <div class="form-group">
-          <%= f.label :genre %>
-          <%= f.select :genre, @genres, prompt: "Choose a genre" %>
-          <%= f.submit "Save", class: 'btn btn-success pull-right' %>
-        </div>
-        
-        <div class="form-group">
-          <%= f.label "Upload Cover Art" %>
-          <%= f.file_field :cover, class: 'btn btn-default' %>
-        </div>
-        
-        <div class="form-group">
-          <%= f.label :published, class: 'checkbox-inline' do %>
-            <%= f.check_box :published %><p class='bg-danger'> Publish - Story is submitted and cannot be edited! </p>
-          <% end %>
+        <div class="row">
+          <div class="form-group col-md-6">
+            <%= f.label "Upload Cover Art" %>
+            <%= f.file_field :cover, class: 'form-control' %>
+          </div>
+          <div class="form-group col-md-4">
+            <%= f.label :genre %>
+            <%= f.select :genre, options_for_select(@genres), {prompt: "Choose a genre"}, class: 'form-control'  %>
+          </div>
+          <div class="form-group col-md-2">
+              <%= f.label :price %>
+              <%= f.number_field :price, class: 'form-control', placeholder: "Free" %>
+          </div>
         </div>
+        <%= f.submit "Save", class: 'btn btn-success' %>
+        <!--<div class="form-group">-->
+        <!--  <%= f.label :published, class: 'checkbox-inline' do %>-->
+        <!--    <%= f.check_box :published %><p class='bg-danger'> Publish - Story is submitted and cannot be edited! </p>-->
```

Comment:
remove some of the commented out stuff too when not using u will find in git history when needed